pages.fr/sunos/dmesg.md

```diff
@@ -1,16 +1,16 @@
 # dmesg
 
-> Ecrire les messages du kernel sur la sortie standard.
-> Plus d'informations : <https://www.unix.com/man-page/sunos/1m/dmesg>.
+> Écrire les messages du noyau sur la sortie standard.
+> Plus d'information: <https://www.unix.com/man-page/sunos/1m/dmesg>.
```

Comment:
I think the plural form ("informations")  is OK.
In all cases, keep the space before the semi-column:
```suggestion
> Plus d'information : <https://www.unix.com/man-page/sunos/1m/dmesg>.
```

Reply:
Our standard is `Plus d'informations :` from here https://github.com/tldr-pages/tldr/blob/main/scripts/set-more-info-link.py